spec/services/sandbox_seed_data/unfunded_mentors_spec.rb

```diff
@@ -0,0 +1,39 @@
+RSpec.describe SandboxSeedData::UnfundedMentors do
+  let(:instance) { described_class.new }
+  let(:environment) { "sandbox" }
+  let(:logger) { instance_double(Logger, info: nil, "formatter=" => nil, "level=" => nil) }
+  let!(:school_partnerships) { FactoryBot.create_list(:school_partnership, 5) }
+
+  before do
+    allow(Logger).to receive(:new).with($stdout) { logger }
+    allow(Rails).to receive(:env) { environment.inquiry }
+
+    stub_const("#{described_class}::MIN_UNFUNDED_MENTORS_PER_LP", 2)
+  end
+
+  describe "#plant" do
+    subject(:plant) { instance.plant }
+
+    it { expect { plant }.to change(Teacher, :count).by(20) }
```

Comment:
Is it worth querying the unfunded mentors using the `Query` service for each LP here?